src/StochasticStyles/spawning.jl

```diff
@@ -361,18 +361,18 @@ Base.@kwdef struct DynamicSemistochastic{T,S<:SpawningStrategy} <: SpawningStrat
     abs_threshold::T = Inf
 end
 
-@inline function spawn!(s::DynamicSemistochastic, w, offdiags::AbstractVector, add, val, boost)
+@inline function spawn!(s::DynamicSemistochastic, w, column::AbstractVector, add, val, boost)
     # assumes that s.strat.threshold is defined
     # special-case substrategies that don't fit the pattern?
-    thresh = min(s.abs_threshold, length(offdiags))
+    thresh = min(s.abs_threshold, length(column))#length might not be known
```

Comment:
If you want this to be the same as before, it would need to be `length(column) - 1`. It might be more readable to have `num_offdiagonals` (or maybe `max_offdiagonals` similar to what Joachim mentioned in #312 ) defined on columns, especially if we'll keep it around for Hamiltonians anyway.